content/Software Support/IDE Settings/Change-the-language-in-Arduino-IDE.md

```diff
@@ -66,6 +62,8 @@ Arduino IDE translations have been made through community contributions, and not
 * [Arduino IDE 1 translation project](https://www.transifex.com/mbanzi/arduino-ide-15/)
 * [Arduino IDE 2 translation project](https://www.transifex.com/arduino-1/ide2/)
 
+Note that modifying the language preferences will not impact the Serial Monitor's output language, which is set to English as default.
+
```

Comment:
```suggestion
```

This statement was nonsensical. Serial Monitor doesn't have an "output language". It simply outputs whatever data it receives from the serial port.

Reply:
>The goal here is to explain that the changes in the Preferences menu will not affect the messages shown in the Output view

Then you must use the correct terminology. "Serial Monitor" != "Output" view. They might share the same bottom panel of the IDE, but they are two completely different things.

>  Although the output language of some tools is set by the system settings, other messages are still presented in English (messages about storage space, etc), even when the system's language has changed.

Arduino IDE doesn't have any control over the localization of 3rd party compilation and upload tools like AVRDUDE so the IDE's "Language" preference won't have any effect on those tools.

A change to Arduino IDE's "Language" preference will also be applied to the localization of Arduino CLI, which is the source of some of the messages that are printed in the "Output" view. The localization configuration change is only applied to Arduino CLI after restarting Arduino IDE.

Reply:
Thanks for the clarification @per1234.

Reply:
@per1234 I believe https://github.com/arduino/help-center-content/commit/8e2aee19cadcad2951ec44a812813dd5dfcc3fca resolves this. Can you confirm and approve the changes if you agree?